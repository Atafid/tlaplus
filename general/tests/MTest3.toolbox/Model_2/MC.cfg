\* CONSTANT definitions
CONSTANT
WhichTest <- const_124951154319217000
SPECIFICATION spec_124951154319218000
\* Generated at Wed Aug 05 15:32:23 PDT 2009