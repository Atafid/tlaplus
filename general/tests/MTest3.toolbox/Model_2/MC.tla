---- MODULE MC ----
EXTENDS MTest3, TLC

\* CONSTANT definitions
const_124951154319217000 == 
2
----

\* Specification formula
spec_124951154319218000 ==
x=0 /\[][ FALSE ]_<<x>> 
----
====
\* Generated at Wed Aug 05 15:32:23 PDT 2009