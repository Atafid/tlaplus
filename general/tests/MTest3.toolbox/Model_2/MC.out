TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest3.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module MTest3
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting...
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2105:1 @!@!@
Evaluating assumption line 11, col 8 to line 14, col 26 of module MTest3 failed.
Attempted to check equality of integer 2 with non-integer:
"b"
@!@!@ENDMSG 2105 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished.
@!@!@ENDMSG 2186 @!@!@
